RGBLED/CS/README.md

```diff
@@ -0,0 +1,270 @@
+  
+This is a Universal Windows application 
+
+### Generate an app package
+
+Steps to follow :
+
+ * In Solution Explorer, open the solution for your application project.
+ * Right-click the project and choose Publish->Create App Packages (before Visual Studio 2019 version 16.3, the Publish menu is named Store).
+ * Select Sideloading in the first page of the wizard and then click Next.
+ * On the Select signing method page, select whether to skip packaging signing or select a certificate for signing. You can select a certificate from your local certificate store, select a certificate file, or create a new certificate. For an MSIX package to be installed on an end user's machine, it must be signed with a cert that is trusted on the machine.
+ * Complete the Select and configure packages page as described in the Create your app package upload file using Visual Studio section.
+
+ If you need guidance click Link: [here](https://docs.microsoft.com/en-us/windows/msix/package/packaging-uwp-apps#generate-an-app-package).  
+  
+### Install your app package using an install script
+
+Steps to follow :
+ * Open the *_Test folder.
+ * Right-click on the Add-AppDevPackage.ps1 file. Choose Run with PowerShell and follow the prompts.
+ * When the app package has been installed, the PowerShell window displays this message: Your app was successfully installed.
+
+ If you need guidance click Link: [here](https://docs.microsoft.com/en-us/windows/msix/package/packaging-uwp-apps#install-your-app-package-using-an-install-script).  
+  
+ Click the Start button to search for the app by name, and then launch it.
+
+If you are using UPBOARD, you have to setup the BIOS GPIO configuration.
+
+### BIOS Settings for UPBOARD
+
+Steps to follow:
+ 
+(1)	After power on the Upboard, Press Del or F7 to enter the BIOS setting.
+ 
+(2)	Under the "Boot -> OS Image ID" Tab:
+    Select "Windows 10 IoT Core".
+ 
+(3)	Under the "Advance" Tab:
+    Select "Hat Configuration" and Click on "GPIO Configuration in Pin Order".
+
+(4) Configure the Pins you are using in the sample as "INPUT" or "OUTPUT".
+
+    In this sample make PIN 3, 5, 7 as "OUTPUT" and initial value as "LOW".
+
+If you need guidance click Link: [here](https://www.annabooks.com/Articles/Articles_IoT10/Windows-10-IoT-UP-Board-BIOS-RHPROXY-Rev1.3.pdf).
+ 
+### Let's look at the code
+
+First, we get the default GPIO controller and check that it's not null.
+`GpioController.GetDefault()` will return null on platforms that do not contain
+a GPIO controller.
+
+```csharp
+private void Page_Loaded(object sender, RoutedEventArgs e)
+{
+    var gpio = GpioController.GetDefault();
+
+    // Show an error if there is no GPIO controller
+    if (gpio == null)
+    {
+        GpioStatus.Text = "There is no GPIO controller on this device.";
+        return;
+    }
+}
+```
+
+If you are running on raspberry pi uncomment below block of code in MainPage.xaml.cs

```

Comment:
The example is for the UpBoard.  In all other projects references to RPi2, Minnowboard and Dragonboard were removed, so it should probably be removed here too.

Reply:
Riddhi suggested to comment out the raspberry pi code rather than deleting. Let us know if need to remove?